src/utils/GetStyledCalendarElement/GetStyledCalendarElement.js

```diff
@@ -12,13 +12,15 @@ export const container = containerSelector => {
     };
   }
 
+  selectedElement.style.display = "inline-block";
   selectedElement.style.width = "700px";
   selectedElement.style.borderTopLeftRadius = "3px";
   selectedElement.style.borderTopRightRadius = "3px";
   selectedElement.style.border = "1px solid #E1E4E8";
-  selectedElement.style.padding = "10px 0 10px 20px";
-  selectedElement.style.margin = "20px auto";
-  selectedElement.style.fontSize = "10px";
+  selectedElement.style.padding = "10px 20px";
+  selectedElement.style.minWidth = "350px";
+  selectedElement.style.fontSize =
+    "calc(12px + (14 - 12) * ((100vw - 350px) / (1800 - 350)))";
```

Comment:
These CSS calculations are "driving me nuts". :)

Just out of curiosity, what do the numbers refer to? E.g. why `1800 - 350`? Also, does the return value fits all screen sizes?

Reply:
Why does fontsize have to be dynamic? The whole widget isn't dynamic, right?

Reply:
Or maybe it isn't dynamic actually only calculated, but why not just take whatever GH has?

Reply:
@gomorizsolt  This is an older CSS trick, but I like to use you can [read here more](https://css-tricks.com/snippets/css/fluid-typography/)

@thisismydesign The widget fit for the container width, right now I give 700px, but you can change, or overwrite it (for example define it as vw, %, etc...) And this way everyone can set the perfect size, that's why I think the dynamic font size is a good idea, but not sure it will worth it. So we can just give fix 14px font-size to the container, and use em unit in the header, and SVG.

Reply:
Let's keep it simple, we don't currently support other sizes.

Reply:
Ok, right now it is look like this in action:

![Peek 2019-09-10 18-03](https://user-images.githubusercontent.com/18705136/64630419-5b140e00-d3f5-11e9-91bc-73415a367b3c.gif)

---

Now I change it to this:

![Peek 2019-09-10 18-06](https://user-images.githubusercontent.com/18705136/64630577-b6de9700-d3f5-11e9-8905-2032301aacdf.gif)

(Both between ~500-1600px width)

Reply:
Alright, just make sure it's fixed size by default please.

Reply:
Default container size: 700px